rakefile.rb

```diff
@@ -344,6 +373,18 @@ def run_tests(test_assemblies, command, result_dir)
   end
 end
 
+def run_dotnet_tests(test_directories, result_dir)
+  test_directories.each do |test_directory|
+    result_file = File.expand_path(File.join(result_dir, File.basename(test_directory)) + '.TestResults.xml')
+
+    cmd = Exec.new
+    cmd.command = 'dotnet'
+    cmd.working_directory = test_directory
+    cmd.parameters "test -c Release --no-build -xml #{result_file} -nologo -notrait \"explicit=yes\""

```

Comment:
I just wanted to highlight the difference. It's probably fine.
Shadow-copying can make running of non-netstandard FIE-using tests slower. But that's not a big deal either.